packages/cli/src/oclif/commands/migrate.js

```diff
@@ -50,12 +59,15 @@ class MigrateCommand extends BaseCommand {
         name: 'migrate',
         from_version: fromVersion,
         to_version: toVersion,
-        email: user,
+        email: user || account,
+        accountType: flagType,
```

Comment:
I think you need to use `email_type` to match the name on the backend.

```suggestion
        email_type: flagType,
```

Reply:
I'd lean towards releasing it publicly, so that the `user` behavior only does the user (rather than the current behavior, where `user` does the accounts)

Reply:
Done, didn't realise that was the key we were using for the backend. 👍